frontend/src/plugins/userManagement/templates/addUser.hbs

```diff
@@ -24,9 +24,17 @@
       <div class="form-group">
         <label for="role">{{t 'app.role'}}</label>
         <select name="role">
-          {{#each globalData.allRoles.models}}
-            <option value="{{this.attributes._id}}">{{this.attributes.name}}</option>
-          {{/each}}
+          {{#if globalData.hasTenantAdminPermissions}}
+            {{#each globalData.allRoles.models}}
+              {{#ifValueNotEquals this.attributes.name "Super Admin"}}
```

Comment:
This check should be permissions-based, rather than role-based. 
(see [Helpers.ifHasPermissions](https://github.com/adaptlearning/adapt_authoring/blob/develop/frontend/src/core/helpers.js#L191))